build/packages.targets

```diff
@@ -3,7 +3,7 @@
         <MicrosoftBuildPackageVersion Condition="'$(MicrosoftBuildPackageVersion)' == ''">16.8.0</MicrosoftBuildPackageVersion>
         <NewtonsoftJsonPackageVersion Condition="$(NewtonsoftJsonPackageVersion) == ''">9.0.1</NewtonsoftJsonPackageVersion>
         <SystemPackagesVersion>4.3.0</SystemPackagesVersion>
-        <VSFrameworkVersion>17.0.0-preview-2-31221-277</VSFrameworkVersion>
+        <VSFrameworkVersion>17.0.0-previews-1-31318-023</VSFrameworkVersion>
```

Comment:
Was this change intentional?

Reply:
Yes, that's really the crux of this change -- upgrade to a VS interop package version compatible with VS's main branch.

Btw the "previews" token (vs. "preview") is correct. Here's how the IDE team explained it:

> Due to our releasing a partner preview or two before our first public preview, our VS SDK nuget packages built from the VS repo had to apparently downgrade from building preview-2 to preview-1. To address this and satisfy nuget's semver rules to recognize newer from older packages, we added an s to the unstable tag. Please note that previews-* packages are newer than preview-*. So your next VS SDK package update will take you from -preview-2 to -previews-1.

Reply:
31318 is larger than 31221, so I guess so. Especially since dev17 preview 1 hasn't shipped to customers yet, so it doesn't make sense to be on preview 2 already.